components/dashboard/script.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+
+const vars = {
+    REACT_APP_HOSTNAME : 'Gitpod',
+    REACT_APP_DOCS_URL : 'https://www.gitpod.io/docs/',
+    REACT_APP_SOCIAL_URL : 'https://community.gitpod.io/',
+    REACT_APP_FAVICON_192 : './favicon192.png',
+    REACT_APP_FAVICON_256 : './favicon256.png',
+    REACT_APP_FAVICON_512 : './favicon512.png',
+    REACT_APP_ICON : 'favicon192.png',
+    REACT_APP_BRAND_SVG : 'gitpod.svg'
+};
+
+if(!fs.existsSync('.env')) {
+    let data = "";
+    for(const prop in vars) {
+        data += prop + "=" + vars[prop] + "\n";
+    }
+    fs.writeFileSync('.env', data);
```

Comment:
<details>
<summary>:warning: <strong>This code is missing a blank line after the code block and before the next statement or before the comment.</strong> (click for details)</summary>

Consider leaving a single blank line after blocks/group of statements and before the next statement or before the comment unless it's on the first line of a block.  
  
Adding a blank line improves code readability and keeps the style consistent.

**[More details](https://confluence.devfactory.com/x/hnvlFQ)**

## The improvement suggestion in this case
Consider adding a single blank line before the statement or comment.  
> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence(s).
</details>

[//]: # (<a id="7d300481-6520-473c-a470-15e6eef9e1d3" data-finding-id="367360902" href="none" codereview-uuid-reserved-tag data-business-id="05d37d16-d6de-46e5-9cc8-3f767198a252" data-analyzer="js" />)